recipes-alexa/avs-device-sdk/avs-device-sdk.inc

```diff
@@ -0,0 +1,80 @@
+#
+#   Copyright (c) 2019 Amazon.com, Inc.
+#   Copyright (c) 2019 Luxoft Sweden AB
+#
+#   SPDX-License-Identifier: Apache-2.0
+#
+
+SUMMARY = "An SDK for commercial device makers to integrate Alexa directly into connected products."
+HOMEPAGE = "https://developer.amazon.com/en-IN/docs/alexa/avs-device-sdk"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=d92e60ee98664c54f68aa515a6169708"
+
+FILESEXTRAPATHS_prepend:= "${THISDIR}/files:"
+DEPENDS = "curl nghttp2 sqlite3 openssl rapidjson"
```

Comment:
Can you please append instead of assigning?

Reply:
done